src/platforms/android/androidiaphandler.cpp

```diff
@@ -59,15 +78,12 @@ AndroidIAPHandler::AndroidIAPHandler(QObject* parent) : IAPHandler(parent) {
                          sizeof(methods) / sizeof(methods[0]));
     env->DeleteLocalRef(objectClass);
   });
-}
-
-AndroidIAPHandler::~AndroidIAPHandler() {
-  MVPN_COUNT_DTOR(AndroidIAPHandler);
-  QAndroidJniObject::callStaticMethod<void>(
-      "org/mozilla/firefox/vpn/InAppPurchase", "deinit", "()V");
```

Comment:
All good, but you are removing the deinit(). Can you remove it from the java code too?

Reply:
Not sure why github displays it like that, but it's still there :) Just moved up a bit. On the right line 30.